services/web/server/src/simcore_service_webserver/exporter/request_handlers.py

```diff
@@ -88,6 +88,10 @@ async def import_project(request: web.Request):
     # bumping this requests's max size
     # pylint: disable=protected-access
     exporter_settings = get_settings(request.app)
+    assert (  # nosec
+        exporter_settings is not None
+    ), "this call was not expected with a disabled plugin"  # nosec
+
```

Comment:
I would actually raise and error to block the request properly. No assertion.

Reply:
I would also raise an error for `duplicate_project` and `export_projec` in this module if settings are disabled.

Reply:
@GitHK these are request handlers and they are **not** added to the routes when the plugin is disabled.
In any case, if the plugin is disabled and for some reason there is an uexpected call to these functions, exporter_settings is actually going to be None and nothing else will work below ... 
IMO there is no need to put more errors

Reply:
totally forgot about the routes removal.